app/controllers/admin/assignees_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Admin::AssigneesController < AdminController
+  def create
+    assignee = Assignee.new(assignee_params)
+
+    respond_to do |format|
+      if assignee.save
+        format.html { redirect_to admin_application_path(assignee.application), notice: 'Successfully assigned lawyer.' }
```

Comment:
Line is too long. [121/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.